src/components/Dashboard/TodoList.vue

```diff
@@ -796,6 +833,70 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     }
   }
 
+  /** check if the item is actually affiliation invite todo; (not a regular filing item). */
+  isAffiliationInvitation (item): boolean {
+    return item.status === FilingStatus.AFFILIATION_INVITATION_PENDING
+  }
+
+  authorizeAffiliationInvitation (isAuthorized, affiliationInvitationTodo): void {
+    AuthServices.authorizeAffiliationInvitation(
+      this.getAuthApiUrl,
+      this.getIdentifier,
+      affiliationInvitationTodo.affiliationId,
+      isAuthorized
+    )
+      .then()
+      .catch(err => {
+        // eslint-disable-line no-console
+        console.log('failed the call for authorization of affiliation invitation', err)
+      })
+  }
+
+  _buildTodoItemIfFromAffiliationInvitation (affiliationInvitation) {
+    const newTodo: TodoItemIF = {
+      draftTitle: undefined,
+      enabled: true,
+      filingId: -1, // not a filing
+      name: undefined,
+      order: 0,
```

Comment:
So all affiliation invitations have order=0, thus displaying above the other tasks maybe? This feels hacky,

Reply:
Updated, to set order as array size. And as you previously mentioned, I moved the code to load the todos in front of the other items.

Reply:
If you set order as array size then these will appear last (if the list is sorted), but if you load these before the other todos then do you actually want them first?

Reply:
I'll double check with the UX on should they be ordered first, as I do not have the explicit requirement for this page (though on the other pages affiliation invitations were always on top.)

Reply:
OK, I'll leave this with you. Resolving.